exchanges/account/account.go

```diff
@@ -31,6 +31,8 @@ var (
 	errBalanceIsNil                 = errors.New("balance is nil")
 	errNoCredentialBalances         = errors.New("no balances associated with credentials")
 	errCredentialsAreNil            = errors.New("credentials are nil")
+	errOutOfSequence                = errors.New("out of sequence")
+	errUpdatedAtIsZero              = errors.New("updated at is zero")
```

Comment:
🦟 Nitpick: I think that this needs to be `updatedAt` 🙂 
Also can reword to make it clearer this error is an error, since it's going down the pb wire.
```suggestion
	errUpdatedAtIsZero              = errors.New("updatedAt may not be zero")
```